package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "todo_list",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack serve --open",
+    "build": "webpack"
```

Comment:
- [ ] You need to add a production and development configuration, even you have a warning when you run "npm start:

![webpack](https://user-images.githubusercontent.com/60631456/125634046-499f567b-56f0-4947-b190-97cda9ccd48c.PNG)